book/tutorials/sar/sentinel1.ipynb

```diff
@@ -293,43 +293,48 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "written-preference",
-   "metadata": {
-    "tags": [
-     "remove-input"
-    ]
-   },
+   "id": "processed-attempt",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "if not os.path.exists('/tmp/tutorial-data'):\n",
-    "    os.chdir('/tmp')\n",
-    "    os.system('git clone --depth 1 https://github.com/snowex-hackweek/tutorial-data.git')"
+    "#%%bash \n",
+    "\n",
+    "# Datasets used in this tutorial are accessed directly from their online repositories\n",
+    "# But we also made a copy on zenodo in case servers are down, etc:\n",
+    "\n",
+    "#cd /tmp\n",
+    "#wget -nc https://zenodo.org/record/RECORD_ID/files/snowex-hackweek/tutorial-data.zip -O tutorial-data.zip\n",
```

Comment:
I think this line should be removed to reduce confusion, which URL to use

Reply:
good catch! i tried to clean up after myself but missed some things